azuredevops/internal/service/feed/resource_feed.go

```diff
@@ -0,0 +1,211 @@
+package feed
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v7/feed"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v7/webapi"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/client"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils"
+)
+
+func ResourceFeed() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceFeedCreate,
+		Read:   resourceFeedRead,
+		Update: resourceFeedUpdate,
+		Delete: resourceFeedDelete,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				ValidateFunc: validation.StringIsNotWhiteSpace,
+				Required:     true,
+				ForceNew:     true,
+			},
+			"project_id": {
+				Type:         schema.TypeString,
+				ValidateFunc: validation.IsUUID,
+				Optional:     true,
+				ForceNew:     true,
+			},
+			"permanent_delete": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"restored": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceFeedCreate(d *schema.ResourceData, m interface{}) error {
+	permanent_delete := d.Get("permanent_delete").(bool)
+	name := d.Get("name").(string)
+
+	if !permanent_delete && isFeedRestorable(d, m) {
```

Comment:
Create/restore does not need to check `permanent_delete`, it should only be used in delete

Reply:
alright, what about the default, cause right now permanent_delete is set by default as true, should it stay like this? or we should set it to false?